.openshift-interop/README.md

```diff
@@ -0,0 +1,67 @@
+# Openshift CI
+
+This folder contains files, required for OpenShift CI.
+It allows us to verify, that Quarkus works on new version of Openshift.
+
+## How it works
+
+The tests are run regularly (every Monday and the time of writing). Test execution is based on configs: [[1]](https://github.com/openshift/release),[[2]](https://github.com/openshift/release/pull/40279/files).
+
+During the run CI builds an image from the Dockerfile in this folder, and runs some tests from the TS.
```

Comment:
It would be useful to specify which modules are ran.

Reply:
Specified, how these modules are chosen.